tests/object.cpp

```diff
@@ -215,10 +254,10 @@ TEST_CASE("object") {
         }
     }
 
-    CppContext context;
+    TestContext d(r);
```

Comment:
`context` wasn't a good name, but I at least have a general idea of what it meant. `d` on the other hand…